Flow.Launcher.Infrastructure/StringMatcher.cs

```diff
@@ -225,6 +215,52 @@ public MatchResult FuzzyMatch(string query, string stringToCompare, MatchOption
             return new MatchResult(false, UserSettingSearchPrecision);
         }
 
+        private bool IsAcronym(string stringToCompare, int compareStringIndex)
+        {
+            if (IsAcronymChar(stringToCompare, compareStringIndex) || IsAcronymNumber(stringToCompare, compareStringIndex))
+                return true;
+
+            return false;
+        }
+
+        // When counting acronyms, treat a set of numbers as one acronym ie. Visual 2019 as 2 acronyms instead of 5
+        private bool IsAcronymCount(string stringToCompare, int compareStringIndex)
+        {
+            if (IsAcronymChar(stringToCompare, compareStringIndex))
+                return true;
+
+            if (char.IsNumber(stringToCompare[compareStringIndex]) || char.IsDigit(stringToCompare[compareStringIndex]))
+            {
+                return compareStringIndex switch
+                {
+                    int i when i == 0 => true,
+                    int i when char.IsWhiteSpace(stringToCompare[i - 1]) => true,
+                    _ => false,
+                };
+            }
+
+            return false;
+        }
+
+        private bool IsAcronymChar(string stringToCompare, int compareStringIndex)
+        {
+            if (char.IsUpper(stringToCompare[compareStringIndex]) ||
```

Comment:
We don't need to add another test for `compareStringIndex!=0` because this is a shortcut operator. It will not test the later when `compareStringInde==0`.